src/ray/gcs/job_state_accessor_test.cc

```diff
@@ -0,0 +1,85 @@
+#include "ray/gcs/job_state_accessor.h"
```

Comment:
the new convention is to put this file in `src/ray/gcs/test/job_state_accessor_test.cc`

Reply:
got it.